rgw/COSBench/cosbench.py

```diff
@@ -0,0 +1,269 @@
+"""Module to trigger COS Bench workloads."""
+import argparse
+import json
+import logging
+import subprocess
+import sys
+from os.path import dirname, join
+from tempfile import NamedTemporaryFile
+from typing import Dict, Optional, Tuple
+
+from jinja2 import Template
+from yaml import safe_load
+
+APP_DIR = "/opt/cosbench"
+LOG = logging.getLogger(__name__)
+
+
+def exec_command(cmd: str) -> subprocess.CompletedProcess:
```

Comment:
Can we remove this method defination and use existing method for executing commands
ref: https://github.com/red-hat-storage/ceph-qe-scripts/blob/2006523432bdea5dc9f8df5f988b66db6cd10108/rgw/v2/utils/utils.py#L20

Reply:
The module structure prevents us from using any definitions outside of `COSBench`... this is the reason why I have redefined the method.